src/Controllers/tasks.ts

```diff
@@ -50,8 +50,8 @@ export const deleteATask = async (req: express.Request, res: express.Response) =
     try {
         const {id} = req.params;
         await deleteTaskByID(id);
-        const updatedTasks = await getTasks();
-        return res.status(200).json(updatedTasks);    
+        const tasks = await getTasks();
```

Comment:
most likely you were sending a json object

updatedTasks: Task[] instead of tasks: Task[], so on the FE when you tried to update tasks, that object it didnt exist, only updatedTasks object existed. But I am not 100% sure I would have to look into the request data to really confirm this